frontend/src/components/MeetingRoom/DeviceSettingsMenu/DeviceSettingsMenu.spec.tsx

```diff
@@ -111,7 +112,9 @@ describe('DeviceSettingsMenu Component', () => {
       const outputDevicesElement = screen.getByTestId('output-devices');
       await waitFor(() => expect(outputDevicesElement.children).to.have.length(3));
       expect(outputDevicesElement.firstChild).toHaveTextContent('System Default');
-      expect((outputDevicesElement.firstChild as HTMLOptionElement).selected).toBe(true);
+      expect((outputDevicesElement.firstChild as Element).classList.contains('Mui-selected')).toBe(
+        true
+      );
```

Comment:
Note: this was returning `undefined` after an update and needed to be changed on how we access the `selected` property.